packages/cyberstorm/src/utils/type_guards.ts

```diff
@@ -3,3 +3,10 @@ export const isRecord = (obj: unknown): obj is Record<string, unknown> =>
 
 export const isStringArray = (arr: unknown): arr is string[] =>
   Array.isArray(arr) && arr.every((s) => typeof s === "string");
+
+export const isNode = (e: EventTarget | null): e is Node => {
+  if (!e || !("nodeType" in e)) {
```

Comment:
Nitpick. I suspect you've fallen victim to one of the classic blunders, namely only fixing the line mentioned in a comment and disregarding the context. Now that this no longer throws an error, it would make more sense to be just
```
export const isNode = (e: EventTarget | null): e is Node =>
  e !== null && "nodeType" in e;
```